ci-operator/config/openshift/cluster-api-actuator-pkg/openshift-cluster-api-actuator-pkg-master.yaml

```
@@ -168,6 +168,20 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-gcp
+- always_run: false
+  as: e2e-gcp-operator-pre
```

For consistency with other periodic pre jobs that we have
```suggestion
  as: e2e-gcp-periodic-pre
```

---

Ops, thanks. Fixed!